provision/rosa-cross-dc/Taskfile.yaml

```diff
@@ -12,7 +12,10 @@ vars:
   ROUTE53_DIR: "{{.ROOT_DIR}}/../aws/route53"
   RDS_DIR: "{{.ROOT_DIR}}/../aws/rds"
   KC_DIR: "{{.ROOT_DIR}}/../openshift"
-  RS_HOT_ROD_PASSWORD:
+  ANSIBLE_DIR: "{{.ROOT_DIR}}/../../ansible"
+  PYTHON_DIR: "{{.ROOT_DIR}}/../../benchmark/src/main/python"
+  BENCHMARK_DIR: "{{.ROOT_DIR}}/../../benchmark/src/main/content/bin"
+  KEYCLOAK_MASTER_PASSWORD:
```

Comment:
@pruivo this change seems to be failing with the below error

```
{kakella@kakella:rosa-cross-dc_20231005-11:50:53]0}$ task
task: Task "common:env" is up to date
task: [common:mvnw] bash -c "./mvnw -v"
[common:mvnw] Apache Maven 3.8.4 (9b656c72d54e5bacbed989b64718c159fe39b537)
[common:mvnw] Maven home: /home/kakella/.m2/wrapper/dists/apache-maven-3.8.4-bin/52ccbt68d252mdldqsfsn03jlf/apache-maven-3.8.4
[common:mvnw] Java version: 17.0.8, vendor: Red Hat, Inc., runtime: /usr/lib/jvm/java-17-openjdk-17.0.8.0.7-1.fc38.x86_64
[common:mvnw] Default locale: en_US, platform encoding: UTF-8
[common:mvnw] OS name: "linux", version: "6.4.15-200.fc38.x86_64", arch: "amd64", family: "unix"
task: Task "common:dataset" is up to date
task: Task "common:datasetprovider" is up to date
task: [create-aurora] echo "Deploying Aurora Database (kakella) in region us-east-1"
[create-aurora] Deploying Aurora Database (kakella) in region us-east-1
task: [create-aurora] ../aws/rds/aurora_create.sh
[create-aurora] Aurora instance 'kakella-instance-1:us-east-1' already exists
task: [fetch-aurora-endpoint] ../aws/rds/aurora_endpoint.sh > .task/aurora-endpoint-kakella
task: Failed to run task "default": template: :1:25: executing "" at <.KEYCLOAK_MASTER_PASSWORD>: KEYCLOAK_MASTER_PASSWORD is not a method but has arguments

```

Reply:
and I have confirmed that the `rosa-cross-dc` task from the main branch works just fine without this change and completes a successful deployment of cross-dc on a ROSA ocp.

Reply:
should be fixed now